components/docs-chef-io/content/automate/reusable/md/restore_troubleshooting.md

```diff
@@ -19,3 +19,173 @@ Try these steps if Chef Automate returns an error while restoring data.
+             "type" : "fs",
+         "settings" : {
+         "location" : "/mnt/automate_backups/opensearch/automate-elasticsearch-data/chef-automate-es6-ingest-service"
+               }
+            },
+         "chef-automate-es6-automate-cs-oc-erchef" : {
+         "type" : "fs",
+         "settings" : {
+         "location" : "/mnt/automate_backups/opensearch/automate-elasticsearch-data/chef-automate-es6-automate-cs-oc-erchef"
+               }
+            }
+         }
+        ```
+
+         - If the pre string in the `location` is not match with `backup_mount`, then we need to to delete the existing snapshots. use below script to delete the snapshot from the one of the automate frontend node.
+
+         ```sh
+            snapshot=$(curl -XGET http://localhost:10144/_snapshot?pretty | jq 'keys[]')
+            for name in $snapshot;do
+	            key=$(echo $name | tr -d '"')
+               curl -XDELETE localhost:10144/_snapshot/$key?pretty
+            done
+         ```
+
+         - The above scritp requires the `jq` needs to be installed, You can install from the airgap bundle, please use command on the one of the automate frontend node to locate the `jq` package.
+
+         ```sh
+         ls -ltrh /hab/cache/artifacts/ | grep jq
+
+         -rw-r--r--. 1 ec2-user ec2-user  730K Dec  8 08:53 core-jq-static-1.6-20220312062012-x86_64-linux.hart
+         -rw-r--r--. 1 ec2-user ec2-user  730K Dec  8 08:55 core-jq-static-1.6-20190703002933-x86_64-linux.hart
+         ```
+
+         - In case of multiple `jq` version, then install the latest one. use the below command to install the `jq` package to the automate frontend node
+
+         ```sh
+         hab pkg install /hab/cache/artifacts/core-jq-static-1.6-20190703002933-x86_64-linux.hart -bf
+         ```
+
+   2. Below steps for object storage as a backup option
+
+      - While at the time of deployment `backup_config` will be `object_storage`
+      - To use the `object_storage`, we are using below template at the time of deployment
+
+      ```sh
+         [object_storage.config]
+          google_service_account_file = ""
+          location = ""
+          bucket_name = ""
+          access_key = ""
+          secret_key = ""
+          endpoint = ""
+          region = ""
+      ```
+
+      - If you configured pre deployment, then we are good
+      - If you want to change the `bucket` or `base_path`, then use below template for Frontend nodes
+
+      ```sh
+      [global.v1]
+        [global.v1.external.opensearch.backup.s3]
+            bucket = "<BUCKET_NAME>"
+            base_path = "opensearch"
+         [global.v1.backups.s3.bucket]
+            name = "<BUCKET_NAME>"
+            base_path = "automate"
+      ```  
+
+      - You can choose any value for the variable `base_path`. `base_path` patch is only required for the frontend node.
+      - Use the command  to apply the above template `chef-automate config patch frontend.toml --fe`
```

Comment:
- Use the command to apply the above template `chef-automate config patch frontend.toml --fe`